internal/report/report.go

```diff
@@ -148,14 +192,29 @@ func newPolicyReportResult(policy policiesv1.Policy, resource unstructured.Unstr
 		Kind:    constants.KubewardenKindAdmissionPolicy,
 	}:
 		name = "ap-" + policy.GetName()
-	default:
-		// this never happens
+	default: // this never happens
 		log.Fatal().Msg("no policy found when creating PolicyReportResult")
 	}
 
 	time := metav1.Now()
 	timestamp := *time.ProtoTime()
 
+	var severity v1alpha2.PolicyResultSeverity
+	var scored bool
+	if policy.GetPolicyMode() == policiesv1.PolicyMode(policiesv1.PolicyModeStatusMonitor) {
+		scored = true
+		severity = SeverityInfo
+	}
```

Comment:
Looking into kyverno, this information seems to be taken from an annotation of the policy. For example, looking at [this](https://kyverno.io/policies/other/restrict-secrets-by-label/restrict-secrets-by-label/) policy, you can see the following interesting annotations being specified by the admin:

```yaml
  annotations:
    # We could generate that via kwctl scaffold, using the metadata of the policy 
    policies.kyverno.io/title: Restrict Secrets by Label
    # The meaning of this field is not obvious to me right now, we should document ourselves
    policies.kyverno.io/category: Other
    # This should be set by the admin manually
    policies.kyverno.io/severity: medium
    # AFAIK this is shown by the policy report web UI, isn't it? I think we have that defined inside of the artifacthub metadata, don't we?
    policies.kyverno.io/subject: Pod, Secret
    # We have that, but I would not add it to the annotations of a policy, this would just clutter etcd IMHO
    policies.kyverno.io/description: >-
      Secrets often contain sensitive information and their access should be carefully controlled.
      Although Kubernetes RBAC can be effective at restricting them in several ways,
      it lacks the ability to use labels on referenced entities. This policy ensures
      that only Secrets not labeled with `status=protected` can be consumed by Pods. 
```

To come back to this PR, I would change this code to read the severity from the policy annotation with key `policies.kubewarden.io/severity` (or something like that)

Reply:
Agree, sounds better. I suppose it's ok to schedule that after this PR, as we need to change the CRDs definitions on kubewarden-controller.

Reply:
>  AFAIK this is shown by the policy report web UI, isn't it? I think we have that defined inside of the artifacthub metadata, don't we?
>    policies.kyverno.io/subject: Pod, Secret

Yes. I think this doesn't need additional work. For the PolicyReportResult subject, it is generated with the ObjectReference i has, which I think is a better fit.


For the rest, I have opened https://github.com/kubewarden/kubewarden-controller/issues/461 and added it to the epic, please have a look.